tommaso/es2-4.sql

```diff
@@ -0,0 +1,7 @@
+SELECT Tutors.id, Tutors.first_name, Tutors.last_name
+FROM Tutors
+LEFT JOIN TutorsAssignment
+ON Tutors.id = TutorsAssignment.tutor_id
+WHERE year_of_experience>=5
+GROUP BY id
+HAVING COUNT(id) < 2;
```

Comment:
qui dev'essere "= 1", altrimenti anche quando un tutor non lavora in nessun corso viene elencato